server/src/state/rooms.ts

```diff
@@ -39,7 +34,12 @@ function isValidRoomName(name: string): VoidResult {
   return Success();
 }
 
-export function createRoom(id: string, name: string, roomName: string, langs?: string[]): Result<{ room: Room }> {
+export function createRoom(
+  id: string,
+  name: string,
```

Comment:
Are these the id and name of the player that created the room? If yes, I think they should be named as such.